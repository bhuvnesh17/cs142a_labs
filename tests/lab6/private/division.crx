// div

func main() : void {
  var a : int;
  var b : int;
  let a = 5;
  let b = 2;
  ::printInt(a/b);
  ::println();
  ::printInt(b/a);
  ::println();

  var c : float;
  var d : float;
  let c = 5.5;
  let d = 1.5;
  ::printFloat(c/d);
  ::println();
  ::printFloat(d/c);
  ::println();
}

