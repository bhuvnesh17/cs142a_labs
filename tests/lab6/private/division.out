2
0
3.66666675
0.27272728
